README.md

```diff
@@ -1,23 +1,46 @@
 # Learning locker docker images
 
-This repository contains the `Dockerfile` to build images for [learning locker](https://github.com/LearningLocker/learninglocker), a LRS to store and analyze xAPI statements.
+This repository contains the `Dockerfile` to build images for:
 
-We follow the learning locker's tag, so an image will have the same tag you can found in learning locker repository.
+- [learning locker](https://github.com/LearningLocker/learninglocker), a LRS to store and analyze xAPI statements.
+- [xapi-service](https://github.com/LearningLocker/xapi-service), a service for xAPI develop by learning locker.
+
+We follow the learning locker's tag, so an image will have the same tag you can found in learning locker and xapi-service repositories.
 
 ## Building a learning locker image
 
-To build a learning locker image, you have to pick the right tag to use and then build it using the docker command, Here with the tag `v2.6.2:
+To build a learning locker image, you have to pick the right [tag](https://github.com/LearningLocker/learninglocker/tags) to use and then build it using the docker command, Here with the tag `v2.6.2`:
+
+```bash
+$ docker build -t fundocker/learninglocker:v2.6.2 --build-arg LL_VERSION="v2.6.2" learning/
+```
+
+## Building a xapi-service image
+
+To build a xapi-service image, you have to pick the right [tag](https://github.com/LearningLocker/xapi-service/tags) to use and then build it using the docker command, Here with the tag `v2.2.15`:
 
 ```bash
-$ docker build -t fundocker/learninglocker:v2.6.2 --build-arg LL_VERSION="v2.6.2" .
+$ docker build -t fundocker/xapi-service:v2.2.15 --build-arg VERSION="v2.2.15" xapi/
 ```
 
+## How to use the CI
+
+The CI configured with this repository will test if the image can be built correctly but is also able to push images to docker hub when we tag this repository.
+
+Here are the step to follow to push a new image:
+
+- Edit file `.circleci/releases.sh` and change the tag version corresponding to the image you want to push.
+- Commit your changes and once merged into master you will be able to tag a new version.
+
+Tags must follow those pattern depending the image you want to push:
+
+- `learninglocker-v.*` if you want to push a new learning locker image. _eg_: `learninglocker-v2.6.2`,
+- `xapi-service-v.*` if you want to push a new xapi-service image. _eg_: `xapi-service-v2.2.15`.
+
 ## Use learning locker image
 
 You have to configure your learning locker image using environment variables. You will found all available variables in the [project documentation](http://docs.learninglocker.net/guides-configuring/#learning-locker-application).
 
-You also have to use the [xapi-service](https://github.com/LearningLocker/xapi-service) provided by learning locker. They also provide a docker image you can use.
-
 There is no entrypoint nor command configured in this image. You will have to declare them when you run your container. Here are the commands to use:
```

Comment:
```suggestion
There is no entrypoint nor command configured in this image. You will have to declare them explicitly when you run your container. A list of commands to use _per_ service follows:
```

Reply:
weird I don't have the whitespace at the beginning in local...

Reply:
Have you seen the rest of this suggestion (we don't care about this space)?